blog/rollkit-the-first-sovereign-rollup-framework.md

```diff
@@ -0,0 +1,74 @@
+---
+head:
+  - - meta
```

Comment:
I think this section should be in the format of:

```yaml
---
head:
  - meta:
      name: title
      content: "Rollkit: The First Sovereign Rollup Framework"
  - meta: 
      name: description
      content: Launch a sovereign, customizable blockchain as easily as a smart contract
```

wdyt?

Reply:
it looks like it's being pulled in correctly from the existing blog, you can preview the metadata here

https://www.opengraph.xyz/url/https%3A%2F%2Frollkit.dev%2Fblog%2Fsovereign-rollups-on-bitcoin